servers/republik/script/pot/add_pot_data.js

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env node
+require('@orbiting/backend-modules-env').config()
+const Promise = require('bluebird')
+
+const PgDb = require('@orbiting/backend-modules-base/lib/PgDb')
+
+Promise.props({
+  pgdb: PgDb.connect()
+}).then(async (connections) => {
+  const { pgdb } = connections
+
+  if (await pgdb.public.packages.findOne({ name: 'DONATE_POT' })) {
+    throw new Error('package DONATE_POT already exists, aborting!')
+  }
+
+  const txn = await pgdb.transactionBegin()
+  try {
+    const now = new Date()
+
+    const crowdfunding = await txn.public.crowdfundings.findOne({
+      name: 'LAUNCH'
+    })
+    const company = await txn.public.companies.findOne({
+      name: 'PROJECT_R'
+    })
+
+    const membershipRewardIds = await txn.public.rewards.find({
+      type: 'MembershipType'
+    })
+      .then(rows => rows.map(r => r.id))
+
+    const givePkg = await txn.public.packages.findOne({
+      name: 'ABO_GIVE',
+      crowdfundingId: crowdfunding.id,
+      companyId: company.id
+    })
+
+    // copy ABO_GIVE option with accessGranted
+    const existingPkgOption = await txn.public.packageOptions.updateAndGetOne(
+      {
+        packageId: givePkg.id,
+        rewardId: membershipRewardIds,
+        accessGranted: false
+      },
+      {
+        minAmount: 0,
+        updatedAt: now
+      }
+    )
+    delete existingPkgOption.id
+    const givePkgOption = await txn.public.packageOptions.insertAndGet({
+      ...existingPkgOption,
```

Comment:
Depending on business logic, as far as I can tell we will have to lower `defaultAmount` to 0 on either original or copy of package option.